src/components/Header/Header.js

```diff
@@ -3,11 +3,17 @@ import React from 'react';
 import homeButton from './home.png';
 import './Header.css';
 
-const Header = () => {
+const Header = ({goHome}) => {
```

Comment:
I'm very proud at our persistence & struggle to understand prop and how to pass down methods.